src/components/access-type-display/access-type-display.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { KeyValue, NoValue } from '@folio/stripes/components';
+
+import { getAccessTypeId } from '../utilities';
+
+const propTypes = {
+  accessStatusTypes: PropTypes.object.isRequired,
+  model: PropTypes.object.isRequired,
+};
+
+function AccessTypeDisplay({ model, accessStatusTypes }) {
+  const accessTypesRecords = accessStatusTypes.resolver.state.accessTypes.records;
+  const hasAccessTypes = !!Object.keys(accessTypesRecords).length;
+  const accessTypeId = getAccessTypeId(model);
+
+  if (!hasAccessTypes) {
+    return null;
+  }
```

Comment:
Just a suggestion, I think it could be
```suggestion
  const accessTypesArePresent = !Object.keys(accessTypesRecords).length;
  
  if (accessTypesArePresent) {
    return null;
  }
  
  const accessTypeId = getAccessTypeId(model);
```
To use less exclamation points